test-app/frontend/src/forms/form/Form.tsx

```diff
@@ -0,0 +1,25 @@
+import Form from '@rjsf/material-ui';
+import { JSONSchema7 } from 'json-schema';
+import { useState } from 'react';
+import './Form.scss';
+
+interface IProps {
+  schema: JSONSchema7,
+  onSubmit: (formData: object) => void,
+}
+
+export const CustomForm = ({ schema, onSubmit }: IProps) => {
+  const [formData, setFormData] = useState({});
+  const onChange = (e: { formData: object; }) => {
+    setFormData(e.formData);
+  };
+  return (
```

Comment:
Nit: extra newline here pls

Reply:
I get a linting error when I try to do this :/

Reply:
ooph. thats embarassing for me